core/src/main/java/org/elasticsearch/index/mapper/FieldNamesFieldMapper.java

```diff
@@ -179,11 +181,20 @@ public boolean isEnabled() {
         }
 
         @Override
```

Comment:
Should you override `parseCreateField` to throw an IllegalStateException or something? I don't think we use it any more.

Reply:
We didn't use it until I pushed the latest commits which put the BWC code in to make sure `_field_names` is still used for indexes created before 6.1. I will make sure I change `parseCreateField` to throw an exception when I remove the BWC code though. Thanks for pointing this out.